lib/plugins/aws/package/compile/functions.js

```diff
@@ -216,6 +216,30 @@ class AwsCompileFunctions {
         artifactFilePath = path.join(this.serverless.serviceDir, '.serverless', artifactFileName);
       }
 
+      const runtimeManagement = this.provider.resolveFunctionRuntimeManagement(
+        functionObject.runtimeManagement
+      );
+      if (runtimeManagement.mode) {
+        const rawRuntimeManagement = runtimeManagement.mode;
+        let convertedRuntimeManagement;
+
+        if (rawRuntimeManagement === 'onFunctionUpdate') {
+          convertedRuntimeManagement = 'Function update';
+        } else {
+          convertedRuntimeManagement =
+            rawRuntimeManagement[0].toUpperCase() + rawRuntimeManagement.substr(1);
+        }
```

Comment:
For better clarity let's maybe rely on _map_, so we resolve CW value via `cwRuntimeOnMap.get(configValue)`